docs/mkdocs/docs/developers/paig-client/code-examples/milvus-integration/a-sample-tutorial.md

```diff
@@ -0,0 +1,253 @@
+ - <a href="files/requirements.txt" download="requirements.txt">requirements.txt</a>
+ - <a href="files/create_milvus_collection.py" download="create_milvus_collection.py">create_milvus_collection.py</a>
+ - <a href="files/query_milvus_collection.py" download="query_milvus_collection.py">query_milvus_collection.py</a>
+
+### Start Milvus and Attu server
+!!! note "Note"
+    If you have your own Milvus server running and you want to use that, you can skip this step.
+
+Run the following command in the directory where you placed the files.
+    ```bash
+    docker-compose up -d
+    ```
+
+### Install the required Python packages
+Run the following command to install the required Python packages. 
+
+For this tutorial, we will be installing the `milvus`, `openai` and `paig_client` packages.
+
+- Create a virtual environment and activate it.
+  ```bash
+  python3 -m venv venv
+  source venv/bin/activate
+  ```
+- Install the required packages.
+  ```bash
+  pip install -r requirements.txt
+  ```
+
+### Export your OpenAPI key
+Export your OpenAI API key as an environment variable to use ir further in the code to create the embeddings.
+   ```bash
+   export OPENAI_API_KEY=<your_openapi_key>
+   ```
+
+### Create Milvus collection
+Run the following command to create a Milvus collection with sample data.
+!!! note "Note"
+    If you are using your own Milvus server, make sure to update the `MILVUS_HOST` and `MILVUS_PORT` in the `create_milvus_collection.py` script.
+
+   ```bash
+   python3 create_milvus_collection.py
+   ```
+   
+### Verify the created collection
+To view the collection and the data inside it, open your web browser and navigate to [http://localhost:3000](http://localhost:3000){target="_blank"}. You should see the Attu interface, where you can browse collections, visualize data, and run queries.
+This is how the data will look like below in Attu interface.
+
+![Attu_Collection](images/attu-collection.png){ loading=lazy }
+
+### Understanding the data
+The data in the collection contains the following fields:
+
+- Text: A brief description of the transaction, truncated for brevity.
+- Users: The names of users (sales and department managers) who have access to this record.
+- Groups: The group who has access to this record.
+- Metadata: Additional information stored in JSON format, including the sales person's name, the country of sale, and the department.
+- Vector: The embeddings for the text field, represented as a list of floating-point numbers truncated for brevity.
+
+### Create the users
+In this tutorial, we will be using **rebecca** and **aaron** as users. Please create these users in the PAIG portal following the steps below.
+
+<a href="images/create_user.gif">![Create User Image](images/create_user.gif){ loading=lazy }</a>
+
+- Log in to the PAIG portal and Navigate to **Account > User Management** and click on the **ADD USER** button in the top right corner. 
```

Comment:
do we need to give link about setting up paig-server

Reply:
Added the link, please check